### PR TITLE
新增flow和sequence两种CodeBlock画图功能

### PR DESCRIPTION
**Pull request by zgce**
新增flow和sequence两种CodeBlock画图功能
如在Markdown文档中有 \`\`\`sequence 就会画出序列图，若有 \`\`\`flow会画流程图
flow画图基于  [flowchart.js](http://flowchart.js.org/)
sequence画图基于 [ js-sequence-diagrams](https://bramp.github.io/js-sequence-diagrams/)

### CODE REVIEW

**Comment by volca** [comment]
Thanks for your pull request. I'll review it ASAP.

**Comment by zgce** [comment]
不好意思，我是首次在github上提交pr，在提交后还在master上合入了别的分支持的东西。现在我将master分支回退到刚提交pr的状态，请重新review看一下，感谢！

File: js/runMathJax.js [hunk 1]
```diff
@@ -24,6 +24,16 @@
 
             // Remove div used for MathJax processing
             mathjaxDiv.remove();
+
+            for (i = 1; i <= g_seq_id; ++i) {
```
Comment (volca) [inline js/runMathJax.js]: Please change `i = 1` to `var i = 1`. It should be a local variable.

File: js/runMathJax.js [hunk 2]
```diff
@@ -24,6 +24,16 @@
...
+                var seqid = make_seq_id(i);
+                drawSeq(seqid);
+            }
+
+            for (i = 1; i <= g_flow_id; ++i) {
```
Comment (volca) [inline js/runMathJax.js]: Same comment for line 28

**Comment by zgce** [comment]
I move all diagram function into diagramflowseq.js and wrap the all draw function  into the diagramFlowSeq variable.
All the  function and variable name use camel case name style.
Please check the code again, thk.

**Comment by volca** [comment]
Thanks for your high quality PR. I will push this version to store soon.

Minor changes for final version:

* Change version from v0.6.7.1 to v0.6.8
* Change extension name back to "Markdown Preview Plus"

**Pull request merged**